test/sinks.test.js

```diff
@@ -7,6 +7,8 @@ const Client = require('@asset-pipe/client');
 const AssetServer = require('@asset-pipe/server');
 const AssetSinkFS = require('@asset-pipe/sink-fs');
 const AssetSinkGCS = require('@asset-pipe/sink-gcs');
+const { hashArray } = require('../utils');
+const { endWorkers } = require('../node_modules/@asset-pipe/server/lib/utils');
```

Comment:
https://github.com/asset-pipe/asset-pipe-build-server/blob/64b2aa68b9d2010509cc786bca42d7dfd59c9f6c/lib/main.js#L329-L331 not enough?